sourcegraph/sourcegraph
# bext: make 'Configure on Sourcegraph' button navigate to manage repositories page when on default Sourcegraph URL

Closes [#3066](https://github.com/sourcegraph/sourcegraph/issues/3066)

https://user-images.githubusercontent.com/25318659/155352705-537b6d6b-5394-4af8-ab1d-f7c041882d3d.mp4

| Before | After |
| ---------- | ------ |
| ![browser - Options - OptionsPage - All Options Pages ⋅ Storybook 2022-02-23 17-40-07](https://user-images.githubusercontent.com/25318659/155353911-7962152f-0966-4104-84c1-d4aa3866019e.png) | ![browser - Options - OptionsPage - All Options Pages ⋅ Storybook 2022-02-23 17-41-53](https://user-images.githubusercontent.com/25318659/155353875-95796523-fc43-4cb0-834f-ef296fa12742.png) |

## Test plan

- `sg run bext`
- `SOURCEGRAPH_API_URL=https://sourcegraph.com sg start web-standalone`
- replace `sourcegraphURL` with your local development server URL (_https://sourcegraph.test:3443_ on my machine) [here](https://github.com/sourcegraph/sourcegraph/blob/070afb3a785d3e152db6f68badf7184fd2383a27/client/browser/src/browser-extension/scripts/optionsPage.main.tsx#L162) and [here](https://github.com/sourcegraph/sourcegraph/blob/683a207c174c676aaaf5e09e8d19c10e26692440/client/browser/src/shared/code-hosts/shared/codeHost.tsx#L818)
- make sure you're logged in to Sourcegraph in browser extension and local development server instance
- open private repository which is not synchronized with your Sourcegraph account and:
  - click 'Configure on Sourcegraph' button and check if you were redirected to Sourcegraph manage repositories page _with repository name entered as a filter query_
  - open browser extension popup and check if private repository message is there with a link to manage repositories page _without any filter queries_
